js/models/conversations.js

```diff
@@ -815,28 +815,34 @@
 
     notify: function(message) {
```

Comment:
This was changed to return a promise - but I don't see where we're waiting on the promise.

The two places I see we're calling this (background.js:358, messages.js:508) we're just blowing past it.

Am I missing something?